products/jbrowse-react-linear-genome-view/.storybook/main.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path')
+const spawn = require('cross-spawn')
+const root = require('find-yarn-workspace-root')()
+
+const pgks = spawn.sync('yarn', ['--json', 'workspaces', 'info'])
```

Comment:
is there a source of this file e.g. was it copied from somewhere? can it be referenced?

Reply:
It was inspired by `products/jbrowse-web/rescripts/yarnWorkspacesRescript.js`, I added a comment to that effect.